pymc3_models/models/NaiveBayes.py

```diff
@@ -0,0 +1,283 @@
+import functools as ft
+
+import numpy as np
+import pymc3 as pm
+import scipy.stats
+from sklearn.metrics import accuracy_score
+import theano
+
+from pymc3_models.exc import PyMC3ModelsError
+from pymc3_models.models import BayesianModel
+
+
+__all__ = ["GaussianNaiveBayes"]
+
+
+# Gaussian Naive Bayes
+# -----------------------------------------------------------------------------
+
+class GaussianNaiveBayes(BayesianModel):
+    """
+    Naive Bayes classification built using PyMC3.
+
+    The Gaussian Naive Bayes algorithm assumes that the random variables
+    that describe each class and each feature are independent and distributed
+    according to independent Normal distributions.
+
+    See the documentation of the `create_model` method for details on the model
+    itself.
+    """
+
+    def __init__(self):
+        super(GaussianNaiveBayes, self).__init__()
+
+    def create_model(self):
+        """
+        Creates and returns the PyMC3 model.
+
+        We note :math:`x_{jc}` the value of the j-th element of the data vector :math:`x`
+        conditionned on x belonging to the class :math:`c`. The Gaussian Naive Bayes
```

Comment:
Typo: `conditioned`

Reply:
Thanks! There must be a bunch of typos left, I'll proof-read before re-submitting.